File: C++/selectionSort.cpp
#include <iostream>

using namespace std;

void selectionSort(int *arr, int n){
    int i,j;
    int menor, temp;

    for(i=0;i<n;i++){
        menor=i;
        for(j=i;j<n-1;j++){
            if(arr[j+1]<arr[menor])
                menor=j+1;
        }
        temp = arr[menor];
        arr[menor]=arr[i];
        arr[i]=temp;
    }
}

int main(int argc, char const *argv[]){
    int n, *arr;
    cin >> n;
    arr = new int[n];

    for (int i = 0; i < n; i++)
        cin >> arr[i];

    selectionSort(arr, n);

    cout << "Hello HacktoberFest!" << endl;
    for (int i = 0; i < n; i++)
        cout << arr[i] << ' ';

    cout << endl;

    return 0;
}

File: Go/selection-sort/selectionSort.go
package main

import "fmt"

func selectionSort(arr []int) {
	n := len(arr)

	for i := 0; i < n-1; i++ {
		min := i
		for j := i+1; j < n; j++ {
			if arr[j] < arr[min] {
				min = j
			}
		}
		arr[i], arr[min] = arr[min], arr[i]
	}
}

func main() {
	var n int

	fmt.Scanf("%d", &n)
	arr := make([]int, n)

	for index := 0; index < n; index++ {
		fmt.Scanf("%d", &arr[index])
	}

	selectionSort(arr)
	
	fmt.Printf("Hello Hacktoberfest!\n")
	for index := 0; index < n; index++ {
		fmt.Printf("%d ", arr[index])
	}

}